module/zfs/metaslab.c

```diff
@@ -5495,8 +5495,8 @@ remap_blkptr_cb(uint64_t inner_offset, vdev_t *vd, uint64_t offset,
 	vdev_t *oldvd = vdev_lookup_top(vd->vdev_spa,
 	    DVA_GET_VDEV(&bp->blk_dva[0]));
 	vdev_indirect_births_t *vib = oldvd->vdev_indirect_births;
-	bp->blk_phys_birth = vdev_indirect_births_physbirth(vib,
-	    DVA_GET_OFFSET(&bp->blk_dva[0]), DVA_GET_ASIZE(&bp->blk_dva[0]));
+	BP_SET_PHYSICAL_BIRTH(bp, vdev_indirect_births_physbirth(vib,
+	    DVA_GET_OFFSET(&bp->blk_dva[0]), DVA_GET_ASIZE(&bp->blk_dva[0])));
```

Comment:
Not a problem now, but once BP_SET_PHYSICAL_BIRTH() start using BF64_SET(), the last may call vdev_indirect_births_physbirth() here twice, at least on debug builds, that is not great.  Eternal problem of macros. :(

Reply:
I meant we could move vdev_indirect_births_physbirth() call out of the macro, storing result in temporary variable.